### PR TITLE
HBASE-27846 Introduce hadoop image

### PR DESCRIPTION
**Pull request by ndimiduk**
First commit is #117 as a dependency.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 24s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +0 :ok: |  hadolint  |   0m  0s |  hadolint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 48s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 10s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m  7s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   0m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 15s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 15s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m  9s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m  3s |  hbase-kubernetes-deployment in the patch passed.  |
| +1 :green_heart: |  unit  |   5m 49s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |   8m 37s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/2/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense markdownlint hadolint shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 4ce5d2c9da61 5.4.0-1094-aws #102~18.04.1-Ubuntu SMP Tue Jan 10 21:07:03 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / 6455991 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/2/testReport/ |
| Max. process+thread count | 1275 (vs. ulimit of 5000) |
| modules | C: hbase-kubernetes-deployment . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/2/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-kubernetes-deployment/dockerfiles/hadoop/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# dockerfiles/hadoop
+
+There is a contact that must be maintained between the container image run in the pod and the
```
Comment (ndimiduk) [inline hbase-kubernetes-deployment/dockerfiles/hadoop/README.md:21]: s/contact/contract/

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m  5s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 30s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 16s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m 12s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  4s |  root in HBASE-27827-kubernetes-deployment failed.  |
| +0 :ok: |  mvndep  |   0m  6s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m  3s |  root in the patch failed.  |
| -1 :x: |  mvninstall  |   0m  4s |  hbase-kubernetes-deployment in the patch failed.  |
| -1 :x: |  compile  |   0m  3s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m  3s |  hbase-kubernetes-deployment in the patch failed.  |
| -1 :x: |  javac  |   0m  3s |  root in the patch failed.  |
| -1 :x: |  javac  |   0m  3s |  hbase-kubernetes-deployment in the patch failed.  |
| -1 :x: |  hadolint  |   0m  0s |  The patch generated 3 new + 0 unchanged - 0 fixed = 3 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| -1 :x: |  javadoc  |   0m  3s |  root in the patch failed.  |
| -1 :x: |  javadoc  |   0m  3s |  hbase-kubernetes-deployment in the patch failed.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m  4s |  root in the patch failed.  |
| -1 :x: |  unit  |   0m  3s |  hbase-kubernetes-deployment in the patch failed.  |
| +0 :ok: |  asflicense  |   0m  7s |  ASF License check generated no output?  |
|  |   |   4m 22s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense javac javadoc unit hadolint shellcheck shelldocs markdownlint xml compile |
| uname | Linux 6980881a5115 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / f521479 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/branch-mvninstall-root.txt |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-mvninstall-root.txt |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-mvninstall-hbase-kubernetes-deployment.txt |
| compile | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-compile-root.txt |
| compile | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-compile-hbase-kubernetes-deployment.txt |
| javac | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-compile-root.txt |
| javac | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-compile-hbase-kubernetes-deployment.txt |
| hadolint | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/diff-patch-hadolint.txt |
| javadoc | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-javadoc-root.txt |
| javadoc | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-javadoc-hbase-kubernetes-deployment.txt |
| unit | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-unit-root.txt |
| unit | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/artifact/yetus-precommit-check/output/patch-unit-hbase-kubernetes-deployment.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/testReport/ |
| Max. process+thread count | 35 (vs. ulimit of 5000) |
| modules | C: . hbase-kubernetes-deployment U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/3/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 hadolint=Haskell Dockerfile Linter 2.12.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 30s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 17s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  4s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m  5s |  hbase-kubernetes-deployment in the patch failed.  |
| -1 :x: |  mvninstall  |   0m 18s |  root in the patch failed.  |
| -1 :x: |  mvninstall  |   0m  5s |  hbase-kubernetes-hadoop-image in the patch failed.  |
| +1 :green_heart: |  compile  |   0m 21s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 21s |  the patch passed  |
| -1 :x: |  hadolint  |   0m  0s |  The patch generated 3 new + 0 unchanged - 0 fixed = 3 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 15s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-deployment in the patch passed.  |
| +1 :green_heart: |  unit  |   6m 22s |  root in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  4s |  hbase-kubernetes-hadoop-image in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 15s |  The patch does not generate ASF License warnings.  |
|  |   |  10m 44s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense markdownlint javac javadoc unit xml compile hadolint shellcheck shelldocs |
| uname | Linux 727059f51c48 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / f521479 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/artifact/yetus-precommit-check/output/patch-mvninstall-hbase-kubernetes-deployment.txt |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/artifact/yetus-precommit-check/output/patch-mvninstall-root.txt |
| mvninstall | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/artifact/yetus-precommit-check/output/patch-mvninstall-hbase-kubernetes-deployment_hbase-kubernetes-hadoop-image.txt |
| hadolint | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/artifact/yetus-precommit-check/output/diff-patch-hadolint.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/testReport/ |
| Max. process+thread count | 1263 (vs. ulimit of 5000) |
| modules | C: hbase-kubernetes-deployment . hbase-kubernetes-deployment/hbase-kubernetes-hadoop-image U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/4/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 hadolint=Haskell Dockerfile Linter 2.12.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by ndimiduk** [comment]
Rebased onto #118, which provides all the required updates for `test-patch`.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/5/console in case of problems.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  1s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +0 :ok: |  markdownlint  |   0m  1s |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 27s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 15s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m 11s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   6m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 24s |  the patch passed  |
| -1 :x: |  hadolint  |   0m  0s |  The patch generated 3 new + 0 unchanged - 0 fixed = 3 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m  4s |  hbase-kubernetes-deployment in the patch passed.  |
| +1 :green_heart: |  unit  |   6m 28s |  root in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-hadoop-image in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-testing-image in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 22s |  The patch does not generate ASF License warnings.  |
|  |   |  16m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/5/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs markdownlint javac javadoc unit xml compile |
| uname | Linux 610495f5cc0b 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / f521479 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
| hadolint | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/5/artifact/yetus-precommit-check/output/diff-patch-hadolint.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/5/testReport/ |
| Max. process+thread count | 1262 (vs. ulimit of 5000) |
| modules | C: hbase-kubernetes-deployment . hbase-kubernetes-deployment/hbase-kubernetes-hadoop-image hbase-kubernetes-deployment/hbase-kubernetes-testing-image U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/5/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 hadolint=Haskell Dockerfile Linter 2.12.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/6/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/7/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 25s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 28s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 15s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m 10s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   7m 10s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 23s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 23s |  the patch passed  |
| +1 :green_heart: |  hadolint  |   0m  1s |  There were no new hadolint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 18s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m  4s |  hbase-kubernetes-deployment in the patch passed.  |
| +1 :green_heart: |  unit  |   5m 50s |  root in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  4s |  hbase-kubernetes-hadoop-image in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-testing-image in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 18s |  The patch does not generate ASF License warnings.  |
|  |   |  17m 30s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/7/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs markdownlint javac javadoc unit xml compile |
| uname | Linux b27012640884 5.4.0-1101-aws #109~18.04.1-Ubuntu SMP Mon Apr 24 20:40:49 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / f521479 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/7/testReport/ |
| Max. process+thread count | 1249 (vs. ulimit of 5000) |
| modules | C: hbase-kubernetes-deployment . hbase-kubernetes-deployment/hbase-kubernetes-hadoop-image hbase-kubernetes-deployment/hbase-kubernetes-testing-image U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/7/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 hadolint=Haskell Dockerfile Linter 2.12.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/8/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 16s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ HBASE-27827-kubernetes-deployment Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 26s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  compile  |   0m 15s |  HBASE-27827-kubernetes-deployment passed  |
| +1 :green_heart: |  javadoc  |   0m 11s |  HBASE-27827-kubernetes-deployment passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  6s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   7m  9s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 25s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 25s |  the patch passed  |
| +1 :green_heart: |  hadolint  |   0m  2s |  There were no new hadolint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 19s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m  4s |  hbase-kubernetes-deployment in the patch passed.  |
| +1 :green_heart: |  unit  |   6m  7s |  root in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-hadoop-image in the patch passed.  |
| +1 :green_heart: |  unit  |   0m  5s |  hbase-kubernetes-testing-image in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 20s |  The patch does not generate ASF License warnings.  |
|  |   |  18m 33s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/8/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/119 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs markdownlint javac javadoc unit xml compile |
| uname | Linux 5ca7b5371788 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | HBASE-27827-kubernetes-deployment / f521479 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/8/testReport/ |
| Max. process+thread count | 1264 (vs. ulimit of 5000) |
| modules | C: hbase-kubernetes-deployment . hbase-kubernetes-deployment/hbase-kubernetes-hadoop-image hbase-kubernetes-deployment/hbase-kubernetes-testing-image U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-119/8/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 hadolint=Haskell Dockerfile Linter 2.12.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by ndimiduk** [comment]
Superseded by https://github.com/apache/hbase-kustomize/pull/6

**Pull request closed without merging**